jujugui/static/gui/src/app/assets/javascripts/jujulib/juju.js

```diff
@@ -50,17 +50,22 @@ var module = module;
    * @param failure {function} A callback to be called on failure. Takes
    *     an xhr object as its only parameter.
   */
-  environment.prototype._makeRequest = function(path, method, params, success, failure) {
+  environment.prototype._makeRequest = function(
```

Comment:
Evidently it can; what's our JS line length? I thought it was 80, but this line is now over 80 characters and eslint isn't complaining.